datadog/data_source_datadog_user.go

```diff
@@ -29,16 +30,66 @@ func dataSourceDatadogUser() *schema.Resource {
 					Optional:    true,
 				},
 				// Computed values
+				"created_at": {
+					Description: "The time when the user was created (RFC3339 format).",
+					Type:        schema.TypeString,
+					Computed:    true,
+				},
+				"disabled": {
+					Description: "Indicates whether the user is disabled.",
```

Comment:
```suggestion
					Description: "True if the user is disabled.",
```
Optional nitpick: make it unambiguous what `true` vs `false` means. (If you apply this change, also apply it to user.md below.)